ui/app/AppLayouts/Wallet/popups/swap/SwapModal.qml

```diff
@@ -32,10 +33,21 @@ StatusDialog {
     rightPadding: Style.current.xlPadding
     backgroundColor: Theme.palette.baseColor3
 
+    Connections {
+        target: root.swapInputParamsForm
+        function onFormValuesChanged() {
+            if(root.swapInputParamsForm.isFormFilledCorrectly()) {
```

Comment:
ideally whenever some parameter changes:
1) Loading animations get triggered immediately
2) After 1s we trigger the actual fetch
So perhaps you can introduce a debouncer here? Otherwise a new calculation will be triggered every time the user enters a digit and Paraswap will throttle us.

Reply:
https://github.com/status-im/status-desktop/assets/60327365/0672c877-36ad-4364-8098-c035afbbb693

Hey Ive done it, you can see here loading starts first and then we call fetch only after a second